### ISSUE TITLE
Sass code splitting and take advantage of mixins

### ISSUE THREAD

**Issue by tomscholz** [Dogfalo/materialize#5024]
I think it would make sense to split the `_globals.scss` into smaller files. The `_globals.scss` should only contain global sass, like table, blockqoute styling etc.
Currently the globals file contains:
* Collections
* Breadcrumbs
* Pagination
* Helpers
* Parallax
* Pushpin
* Shadows
* Icon styling
* Progress
* Media

**Comment by Dogfalo** [comment]
I agree some of this might be useful, but is not a priority at the moment.

**Comment by jelofson** [comment]
Agree. It's not super easy to modify things like breadcrumbs, especially the text color (:before).
